src/dsp_tools/commands/project/models/group.py

```diff
@@ -224,10 +224,6 @@ def _toJsonObj_update(self) -> dict[str, Any]:
             tmp["selfjoin"] = self._selfjoin
         return tmp
 
-    def delete(self) -> Group:
```

Comment:
Does this have any connection or did you just stuble upon dead code?

Reply:
when investigating the call hierarchy of `connection.logout()`, I came across some methods that were used in tests only, I think this was one of them